router/v3/channels_test.go

```diff
@@ -617,6 +617,16 @@ func TestHandlers_EditChannelTopic(t *testing.T) {
 			Status(http.StatusBadRequest)
 	})
 
+	t.Run("bad request (over 500 letters)", func(t *testing.T) {
+		t.Parallel()
+		e := env.R(t)
+		e.PUT(path, channel.ID.String()).
+			WithCookie(session.CookieName, commonSession).
+			WithJSON(&PutChannelTopicRequest{Topic: "123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901"}).
```

Comment:
ちなみに、どちらでも良いのですが、`strings.Repeat` という関数があってそちらの方が読みやすいかもしれないです
```suggestion
			WithJSON(&PutChannelTopicRequest{Topic: strings.Repeat("a", 501)}).
```

Reply:
初めて知りました。ありがとうございます！直します